app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,53 @@
+
+module Api::V1
+	class BooksController < ApplicationController
+		before_action :authenticate_user
+		before_action :set_book, only: [:show, :update, :destroy]
+
+		def index
```

Comment:
When i request for books this is what I get -> **Unauthorized**

![image](https://user-images.githubusercontent.com/6015315/62336638-ca980300-b503-11e9-8c35-2ded7f3bbb25.png)

Why? I am not telling the server that "hey I am a logged in user and I want to see the books":

1. I don't know how to tell the server that (I don't have a "token" to show the server that I am an authenticated user).

2. I don't have a token because when I logged in I wasn't given any.